json

ezeelivin opened the pull request:
json

ezeelivin committed:
json

json

moodyblue commented:
Thanks for your contribution. Can you please let me know what is the objective ?

ezeelivin committed:
--reader-bg: var(--vscode-editor-background, #ffffff);
  --reader-fg: var(--vscode-editor-foreground, #1f2328);
  --reader-muted: var(
    --vscode-descriptionForeground,
    color-mix(in srgb, var(--reader-fg) 62%, var(--reader-bg))
  );
  --reader-border: var(
    --vscode-panel-border,
    color-mix(in srgb, var(--reader-fg) 14%, var(--reader-bg))
  );
  --reader-surface: var(
    --vscode-sideBar-background,
    color-mix(in srgb, var(--reader-bg) 96%, var(--reader-fg) 4%)
  );
  --reader-accent: var(--vscode-textLink-foreground, #0969da);
  --reader-mark: color-mix(in srgb, var(--reader-accent) 35%, #ffe082);
  --reader-mark-active: color-mix(in srgb, var(--reader-accent) 55%, #ffd54f);
  --reader-scrollbar-track: color-mix(in srgb, var(--reader-surface) 76%, var(--reader-bg));
  --reader-scrollbar-thumb: color-mix(in srgb, var(--reader-border) 72%, var(--reader-fg) 28%);
  --reader-scrollbar-thumb-hover: color-mix(in srgb, var(--reader-border) 56%, var(--reader-fg) 44%);
  --reader-blockquote-bg: color-mix(in srgb, var(--reader-bg) 93%, var(--reader-accent) 7%);
  --reader-blockquote-border: color-mix(in srgb, var(--reader-accent) 54%, var(--reader-border));
  --reader-blockquote-fg: color-mix(in srgb, var(--reader-fg) 96%, var(--reader-bg) 4%);
  --reader-blockquote-shadow: rgba(15, 23, 42, 0.08);
  --reader-font-size: 14px;
  --reader-line-height: 1.6;
}

body[data-reader-theme="light"] {
  --reader-bg: #ffffff;
  --reader-fg: #1f2328;
  --reader-muted: #5f6c7b;
  --reader-border: #d0d7de;
  --reader-surface: #f6f8fa;
  --reader-accent: #0969da;
  --reader-mark: #fff59d;
  --reader-mark-active: #ffe082;
  --reader-scrollbar-track: #eef1f4;
  --reader-scrollbar-thumb: #c7cdd6;
  --reader-scrollbar-thumb-hover: #aeb7c3;
  --reader-blockquote-bg: color-mix(in srgb, #ffffff 93%, #0969da 7%);
  --reader-blockquote-border: color-mix(in srgb, #0969da 54%, #d0d7de);
  --reader-blockquote-fg: #2f3a46;
  --reader-blockquote-shadow: rgba(15, 23, 42, 0.06);
}

body[data-reader-theme="dark"] {
  --reader-bg: #161b22;
  --reader-fg: #e6edf3;
  --reader-muted: #9ca3af;
  --reader-border: #30363d;
